dataikuapi/dss/admin.py

```diff
@@ -1136,12 +1137,89 @@ def set_definition(self, definition):
             body = definition)
 
 
+class DSSPersonalApiKey(object):
+    """
+    A personal API key on the DSS instance.
+    Do not create this object directly, use :meth:`dataikuapi.DSSClient.get_personal_api_key` instead.
+    """
+    def __init__(self, client, id_):
+        self.client = client
+        self.id_ = id_
+
+    ########################################################
+    # Key description
+    ########################################################
+
+    def get_definition(self):
+        """
+        Get the API key's definition
+        
+        :returns: the personal API key definition, as a JSON object
+        """
+        return self.client._perform_json(
+            "GET", "/personal-api-keys/%s" % (self.id_))
+
+    ########################################################
+    # Key deletion
+    ########################################################
+
+    def delete(self):
+        """
+        Delete the API key
+        """
+        return self.client._perform_empty(
+            "DELETE", "/personal-api-keys/%s" % self.id_)
+
+
+class DSSPersonalApiKeyListItem(dict):
+    """
+    An item in a list of personal API key. 
+    Do not create this object directly, use :meth:`dataikuapi.DSSClient.list_personal_api_keys` or :meth:`dataikuapi.DSSClient.list_all_personal_api_keys` instead.
+    """
+    def __init__(self, client, data):
+        super(DSSPersonalApiKeyListItem, self).__init__(data)
+        self.client = client
+
+    def to_personal_api_key(self):
+        """Gets the :class:`DSSPersonalApiKey` corresponding to this item"""
+        return DSSPersonalApiKey(self.client, self._data["id"])
+
+    @property
+    def id(self):
+        return self._data["id"]
```

Comment:
Testing any properties leads to:
```py
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-4-825eb33a416e> in <module>()
      1 personal_keys = client.list_personal_api_keys(as_type="listitems")
      2 test = personal_keys[3]
----> 3 date = test.created_on
      4 date.isoformat()

/Users/mhoarau/devenv100/dataiku-api-client-python/dataikuapi/dss/admin.py in created_on(self)
   1207     @property
   1208     def created_on(self):
-> 1209         timestamp = self._data["createdOn"]
   1210         return datetime.datetime.fromtimestamp(timestamp / 1000) if timestamp > 0 else None
   1211 

AttributeError: 'DSSPersonalApiKeyListItem' object has no attribute '_data'
```

Reply:
Fixed by the commit pushed by Mickael directly on the PR